packages/core/admin/ee/server/controllers/user.js

```diff
@@ -1,17 +1,44 @@
 'use strict';
 
-const { get } = require('lodash');
+// eslint-disable-next-line node/no-extraneous-require
+const ee = require('@strapi/strapi/lib/utils/ee');
+const _ = require('lodash');
 const { pick } = require('lodash/fp');
-const { ApplicationError } = require('@strapi/utils').errors;
+const { ApplicationError, ForbiddenError } = require('@strapi/utils').errors;
 const { validateUserCreationInput } = require('../validation/user');
+const {
+  validateUserUpdateInput,
+  validateUsersDeleteInput,
+} = require('../../../server/validation/user');
 const { getService } = require('../../../server/utils');
 
 const pickUserCreationAttributes = pick(['firstname', 'lastname', 'email', 'roles']);
 
+const hasAdminSeatsAvaialble = async () => {
+  if (!strapi.EE) {
+    return true;
+  }
+
+  const permittedSeats = ee.seats;
+  if (!permittedSeats) {
+    return true;
+  }
+
+  const userCount = await strapi.service('admin::user').getCurrentActiveUserCount();
+
+  if (userCount < permittedSeats) {
+    return true;
+  }
+};
+
 module.exports = {
   async create(ctx) {
+    if (!(await hasAdminSeatsAvaialble())) {
+      throw new ForbiddenError('License seat limit reached. You cannot create a new user');
+    }
+
     const { body } = ctx.request;
-    const cleanData = { ...body, email: get(body, `email`, ``).toLowerCase() };
+    const cleanData = { ...body, email: _.get(body, `email`, ``).toLowerCase() };
```

Comment:
we don't allow upper case emails in user creation? if so why changing it here?

Reply:
it's a copy paste from ce code I made sure it was consistent an not broken

Reply:
👌